src/components/NavBar/ChainSelector.tsx

```diff
@@ -53,9 +53,7 @@ function getChainsFromEIP155Accounts(accounts?: string[]): SupportedChainId[] {
 function useWalletSupportedChains() {
   const { connector } = useWeb3React()
 
-  const getConnection = useGetConnection()
-
-  const connectionType = getConnection(connector).type
+  const connectionType = getConnection(connector)?.type
```

Comment:
Is the optional operator necessary?

Reply:
You're right it's not necessary, I'm not sure why vscode was complaining about it originally